src/chrome/content/rules/opencaching.de.xml

```diff
@@ -0,0 +1,10 @@
+<!--
+	Main pages support https
+	Sub domains like forum / blog / testing do not support or corrupted https
```

Comment:
Please list them and also the cause (e.g. mismatch, refused...)
blog.opencaching.de
download.opencaching.de
forum.opencaching.de
redmine.opencaching.de
rwd.opencaching.de
wiki.opencaching.de